app/manifest/firefox.json

```diff
@@ -4,5 +4,6 @@
       "id": "webextension@metamask.io",
       "strict_min_version": "68.0"
     }
-  }
+  },
+  "content_security_policy": "script-src 'self' blob:; object-src 'self'"
```

Comment:
we cant allow this

Reply:
The reason we add this is because [react-qr-reader](https://github.com/JodusNodus/react-qr-reader) the QR code reader we are using is creating a web worker to optimize the speed and experience of QR code scanning. So the web worker needs to create a Blob to do it.

related code:https://github.com/JodusNodus/react-qr-reader/blob/521b4ad1910ca24e0071c0502044526a282521e1/src/index.js#L13

Reply:
To resolve this, we think there are two options here:

Option 1: Currently MetaMask uses zxing library(0.8.0) to scan the QR Code but the version is quite old which can't support continuously scanning. (The animated QR Code needs this.) We found the last version(0.18.6) can support continuous scanning, we can do some investigation to see whether it fit our need.

Option2: We forked the `react-qr-reader` and change the code to refactor the web worker to see whether the blob can be deleted，and meanwhile delete the webrtc。

Reply:
For us, we prefer option 1 which will not introduce the `react-qr-reader`.  But meanwhile, it means you have to upgrade the current  @zxing/library to the latest version.

@kumavis @danjm please share your thoughts

Reply:
we will start working on option 1 and will update the PR based on it. But if we think we need a discussion about the two options, we are here for it.

Reply:
`react-qr-reader` has been removed so this CSF is also removed

Reply:
thanks